Donut_Shop.js

```diff
@@ -1,52 +1,81 @@
 
   (function () {
 
-    var topPotDonuts = function(franchise, options) {
+    var TopPotDonuts = function(franchise, options) {
       this.franchise = franchise;
       this.minCustomersPerHour = options.minCustomersPerHour;
       this.maxCustomersPerHour = options.maxCustomersPerHour;
       this.avrgDonutsPerCust = options.avrgDonutsPerCust;
-      this.myArray = [];
+      this.hourlyArray = []; //Stores individual hourlyTotal values for each iteration of loop
+      this.total;  //stores sum of hourlyrray values
     };
 
-   topPotDonuts.prototype.hourlyTotal = function() {
+
+   TopPotDonuts.prototype.hourlyTotal = function() {
     var custPerDay = Math.floor(Math.random() * (this.maxCustomersPerHour - this.minCustomersPerHour) + this.minCustomersPerHour);
       return Math.round(custPerDay * this.avrgDonutsPerCust);
    };
 
-   topPotDonuts.prototype.dailyTotal = function () {
-      var total = 0;
-      for(var i = 0; i <= 11; i++) {
-        total += this.hourlyTotal();
-        this.myArray.push(total);
+   TopPotDonuts.prototype.dailyTotal = function () {
+      this.total = 0;
+      for(var i = 0; i < 11; i++) {
+        this.total += this.hourlyTotal();
+        this.hourlyArray.push(this.total);
       };
-      console.log(this.myArray);
-      return total;
    }
 
 
-    topPotDonuts.prototype.render = function() {
-     var dailyDonuts = this.dailyTotal();
-        var elRows = document.getElementById(this.franchise);
-      for(var i = 0; i <= this.myArray.length; i++) {
-        var td = document.createElement('td');
-        td.textContent = this.myArray[i];
-        elRows.appendChild(td);
-      }
-      td.textContent = dailyDonuts;
-    elRows.appendChild(td);
+    TopPotDonuts.prototype.render = function() {
+      // 1. Info needed from constructor obj: location name, hourly totals, daily totals.
```

Comment:
I know this is a work in progress, so just want to reiterate that comments are also part of that "code smell" category that you'll want to clean up when pushing a final commit for a pull request. Get that habit in place soon if you've not already started with your more recent assignments.